src/Processor/Consumers/CustomsDeclarationsConsumer.cs

```diff
@@ -60,21 +59,18 @@ CancellationToken cancellationToken
         if (alvsValErrors.Length == 0)
             return;
 
-        var processingErrorNotification = new DataApiErrors.ErrorNotification
+        var processingErrorNotification = new DataApiErrors.ProcessingError
         {
             Created = DateTime.UtcNow,
```

Comment:
let's set the `CorrelationId` here, for now we can set it to `customsDeclarationsMessage.ServiceHeader.CorrelationId` & add a comment that we will amend how the correlation identifier is assigned when we have got confirmation of the proposed logic from HMRC